### ISSUE TITLE
New Assignment Emails

### ISSUE THREAD

**Issue by ChasonMindImp** [maltyeva/mindimp-pg#107]
Is the site designed to email students when a new assignment has been posted?

**Comment by maltyeva** [comment]
yes, actually matt and i were going to test this yesterday, but let's do it this afternoon. He was able to post an assignment to my test class with all the teachers, but not to a class with the students. We need to test this with my laptop, so I can run logs. 

What I thiiink this may be related to is the fact that Chinese students often have emails that are just numbers, so it may be catching on something.
